CHANGELOG.md

```diff
@@ -29,6 +29,24 @@ CHANGELOG
 Swift 5.5
 ---------
 
+* [SE-0313][]:
+
+  Declarations inside an actor that would normally by actor-isolated can
+  explicitly become non-isolated using the `nonisolated` keyword. Non-isolated
+  declarations can be used to conform to synchronous protocol requirements:
+
+  ```swift
+  actor Account: Hashable {
+    let idNumber: Int
+    let balance: Double
+
+    nonisolated func hash(into hasher: inout Hasher) { // okay, non-isolated satisfies synchronous requirement
+      hasher.combine(idNumber) // okay, can reference idNumber from outside the let
+      hasher.combine(balance) // error: cannot synchronously access actor-isolated property
```

Comment:
This example does not seem right,

both `idNumber` and `balance` are `let`s and none of them are `nonisolated`.
Either none of them or both would produce an error, right? 🤔

Is it missing `nonisolated` on `idNumber` ?

Reply:
Ah, `balance` should have been a `var`, thank you!